recipe/meta.yaml

```diff
@@ -30,7 +32,7 @@ requirements:
     - libtool                                             # [unix]
     - automake                                            # [unix]
     - autoconf                                            # [unix]
-    - cmake                                               # [win]
+    - cmake 3.25.0                                        # [win]
```

Comment:
Hi @traversaro , why is this pin required?

Reply:
Could you please add a comment?

Reply:
Sorry, I was debugging the build failure and this was the last attempt before I left the keyboard. Indipendently from the run_exports, it seems that there was a build regression of the recipe on Windows from CMake 3.25.0 to 3.25.1 
 . I will clean up the PR as soon as I am back with a laptop.